src/main/java/codeu/controller/ActivityServlet.java

```diff
@@ -1,27 +1,83 @@
 
-
 package codeu.controller;
 
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
 import java.io.IOException;
 
-import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
 import codeu.model.store.basic.UserStore;
+import java.util.List;
+import java.util.UUID;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/*Servlet class responsible for the activity feed page*/
```

Comment:
nitpick:
Add space before and after the comment sentence. Also period at the end of the sentence.
/* Servlet class responsible for the activity feed page. */

Please apply the same change to comments throughout the file .